react/ImageSlider.tsx

```diff
@@ -62,6 +62,7 @@ function ImageSlider(props: ImageSliderProps) {
               link={imageLink}
               maxHeight={height}
               width="100%"
+              preload={idx === 0}
```

Comment:
Shouldn't that be a prop as well? Although I don't see problems with it not being a prop, I think it's weird for this to behave differently from the list context.